Workflow.cpp

```diff
@@ -161,6 +161,36 @@ void Workflow::inlineMethodCalls(AnalysisContextRef ac)
                 log->LogInfo("Structures analyzed in %lu ms", elapsed.count());
 
                 InfoHandler::applyInfoToView(info, bv);
+
+                const auto msgSendFunctions = findMsgSendFunctions(bv);
+                for (auto addr : msgSendFunctions)
+                {
+                    BinaryNinja::QualifiedNameAndType nameAndType;
+                    std::string errors;
+                    std::set<BinaryNinja::QualifiedName> typesAllowRedefinition;
+
+                    // void *
+                    auto retType = BinaryNinja::Confidence<BinaryNinja::Ref<BinaryNinja::Type>>(
+                            BinaryNinja::Type::PointerType(bv->GetAddressSize(),BinaryNinja::Type::VoidType(), 
+                            0));
+
+                    std::vector<BinaryNinja::FunctionParameter> params;
+                    bool varArg = true;
```

Comment:
This variable is only used once; please remove and just substitute its usage for `true`.